### PR TITLE
Tkhasanov/customkeyboard

### PR DESCRIPTION
**Pull request by Jam42**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,53 +1,65 @@
...
 def read_yaml():
     "Reading yaml from file"
     with open('schedule.yaml') as stream:
         data = yaml.load(stream)
     return data
 
-@BOT.message_handler(weekdays)
+
+def check_even():
+    "Asking for type of week"
```
Comment (mazzz1y) [inline bot.py]: Please change comment

File: bot.py [hunk 2]
```diff
@@ -1,53 +1,65 @@
...
+"Schedule bot"
 import os
 import datetime
 import telebot
 import yaml
 
 BOT = telebot.TeleBot(os.environ["TELEGRAM_TOKEN"])
 
-weekdays=['monday', 'tuesday', 'wednesday', 'thursday', 'friday']
+WEEKDAYS = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday']
```
Comment (mazzz1y) [inline bot.py]: need to add all weekdays

File: bot.py (hunk 1 above)
Comment (Jam42) [reply bot.py]: done

File: bot.py (hunk 2 above)
Comment (Jam42) [reply bot.py]: done

**Comment by mazzz1y** [comment]
&#128077;

**Pull request merged**